src/core/evaluator.js

```diff
@@ -2190,7 +2191,14 @@ class PartialEvaluator {
         }
         textState.translateTextMatrix(tx, ty);
 
-        textChunk.str.push(glyphUnicode);
+        // remark(henrywarhurst): Add glyph to the beginning of the string for
```

Comment:
You can summarize this as:

```javascript
// Add the glyph to the beginning of the string for RTL languages,
// and to the end for LTR languages (fixes issue 12460).
```

This is a bit more compact and the common style for comments within the codebase. We don't use a username mention style.